build.sbt

```diff
@@ -92,35 +92,48 @@ getProtoFiles := {
 
     println(s"Downloading and installing SDK$sdkMajor protocol buffer files...")
 
-    val bblfshProto = f"$protoDir%s/gopkg.in/bblfsh"
-    val sdkProto = f"$bblfshProto%s/sdk.$sdkMajor%s"
+    val bblfshProto = s"${protoDir}/github.com/bblfsh"
+    val sdkProto = s"${bblfshProto}/sdk/${sdkMajor}"
 
-    f"mkdir -p $sdkProto%s/protocol" !
+    s"mkdir -p ${sdkProto}/protocol" !
 
-    f"mkdir -p $sdkProto%s/uast" !
+    s"mkdir -p ${sdkProto}/uast/role" !
 
     val unzip_dir = "sdk-" + sdkVersion.substring(1)
 
-    f"curl -SL https://github.com/bblfsh/sdk/archive/$sdkVersion%s.tar.gz" #| "tar xz" #&&
-    f"cp $unzip_dir%s/protocol/generated.proto $sdkProto%s/protocol/" #&&
-    f"cp $unzip_dir%s/uast/generated.proto $sdkProto%s/uast/" #&&
-    f"rm -rf $unzip_dir%s" !
+    s"curl -SL https://github.com/bblfsh/sdk/archive/${sdkVersion}.tar.gz" #| "tar xz" #&&
+    s"cp ${unzip_dir}/protocol/driver.proto ${sdkProto}/protocol/" #&&
+    s"cp ${unzip_dir}/uast/role/generated.proto ${sdkProto}/uast/role" #&&
+    s"rm -rf ${unzip_dir}" !
+
+    println("Done unpacking SDK")
 }
 
 val getLibuast = TaskKey[Unit]("getLibuast", "Retrieve libuast")
 getLibuast := {
     import sys.process._
 
-    val releaseUrl = s"https://github.com/bblfsh/libuast/archive/v$libuastVersion.tar.gz"
-    println(s"Downloading libuast from $releaseUrl")
+    val ghUrl = "https://github.com/bblfsh/libuast"
+    val os = if (System.getProperty("os.name").toLowerCase.contains("mac os x")) "darwin" else "linux"
+    val binaryReleaseUrl = s"${ghUrl}/releases/download/v${libuastVersion}/libuast-${os}-amd64.tar.gz"
+    println(s"Downloading libuast binary from ${binaryReleaseUrl}")
+
+    s"curl -sL ${binaryReleaseUrl} -o libuast-bin.tar.gz" #&&
```

Comment:
For future consideration: We can probably simplify this script if we run it with `set -e` and `set -o pipefail` enabled; that would allow us to drop some of the explicit connectives. I'm not clear how easy that is from the shell runner here, though.

Reply:
I agree. Most probably going to replace this this with either shell or a Makefile later on, that will do this work and call into `./sbt`. But would prefer doing that a bit later, after bare-bone functionality of #84 is in place.

Reply:
Filed https://github.com/bblfsh/client-scala/issues/88 for it not to get lost.